opengemini_client/client_impl.py

```diff
@@ -128,8 +131,25 @@ def ping(self, idx: int):
             return None
         return Error(f"ping openGeminiDB status is {resp.status_code}")
 
-    def query(self, query: Query) -> tuple:
-        return ()
+    def query(self, query: Query):
+
```

Comment:
Redundant line

Reply:
Seems to be left over from conflict resolution。local repo is correct。